fvtest/tril/tril/ilgen.hpp

```diff
@@ -50,7 +50,11 @@ class TRLangBuilder : public TR::IlInjector {
         TRLangBuilder(const ASTNode* trees, TR::TypeDictionary* d)
               : TR::IlInjector(d), _trees(trees) {}
 
+#if __cplusplus <= 199711L // Check compiler support for C++11.
```

Comment:
For changes like this, since override is not mandatory, I think I'd prefer to skip the `#ifdef` and instead simply annotate the function with a comment;

```
bool injectIL() /* override */ ;
```

Reply:
Same applies elsewhere